.github/workflows/scripts/pre_before_install.sh

```diff
@@ -1,18 +1,25 @@
 set -mveuo pipefail
 
 
-update_docker_configuration() {
-  echo "INFO:
-  Updating docker configuration
-  "
+echo "INFO:
+Updating docker configuration
+"
 
-  echo '{
+sudo touch /etc/docker/daemon.json
+echo "$(cat /etc/docker/daemon.json | jq -s '.[0] + {
   "registry-mirrors": ["https://mirror.gcr.io"],
   "insecure-registries" : ["pulp:80"],
   "mtu": 1460
-}' | sudo tee /etc/docker/daemon.json
-  sudo service docker restart
-}
+}')" | sudo tee /etc/docker/daemon.json
+sudo service docker restart
 
+echo "INFO:
+Updateing registries configuration
+"
 
-update_docker_configuration
+sudo touch /etc/containers/registries.conf
+echo "[registries.insecure]
+registries = ['pulp:80']
+[registries.search]
+registries = ['docker.io', 'pulp:80']
+" | sudo tee /etc/containers/registries.conf
```

Comment:
```suggestion
echo "[registries.insecure]
registries = ['pulp:80']
" | sudo tee -a /etc/containers/registries.conf
```

Let's use the registries that were already configured.